docs/developers/docker-node-key.md

```diff
@@ -0,0 +1,100 @@
+---
+sidebar_label: Managing Wallets in Docker
+---
+
+# Docker and `cel-key`
+
+## Prerequisites
+
+First, familiarize yourself with the [guide](../nodes/docker-images.mdx) on how to run `celestia-node` with Docker.
+
+## Running your node
+
+Run the Docker image (in this example, we are using a Light Node):
+
+```bash
+docker run -e NODE_TYPE=light -e P2P_NETWORK=mocha -p 26659:26659 celestia-node celestia light start --core.ip https://rpc-mocha.pops.one --gateway --gateway.addr 127.0.0.1 --gateway.port 26659 --p2p.network mocha
```

Comment:
Is the environment variable `NODE_TYPE=light` needed? We already specify that we want to run a light node as a parameter.

Reply:
Yes, I believe the default in `entry point.sh` is bridge

Reply:
Yeah, user needs to declare it for the node store to be set correctly

```
docker run -e P2P_NETWORK=mocha -p 26659:26659 celestia-node celestia light start --core.ip https://rpc-mocha.pops.one --gateway --gateway.addr 127.0.0.1 --gateway.port 26659 --p2p.network mocha

2023-02-07T15:04:03.114Z	INFO	node	nodebuilder/init.go:19	Initializing Bridge Node Store over '/root/.celestia-bridge-mocha'
2023-02-07T15:04:03.117Z	INFO	node	nodebuilder/init.go:50	Saving config	{"path": "/root/.celestia-bridge-mocha/config.toml"}
2023-02-07T15:04:03.118Z	INFO	node	nodebuilder/init.go:51	Node Store initialized
Error: fslock: error opening file: open /root/.celestia-light-mocha/lock: no such file or directory
```

Reply:
Why do we need the environment variable if we need to provide the node type as a parameter?

Reply:
because the default is set to `bridge` in the `Dockerfile` it does not work if you don't declare the env variable, as shown above

Reply:
(i think)